zulip_botserver/tests/test_server.py

```diff
@@ -108,19 +107,19 @@ def test_wrong_bot_credentials(self, mock_StateHandler: mock.Mock, mock_LoggingE
                 'token': 'abcd1234',
             }
         }
-        # TODO: The following passes mypy, though the six stubs don't match the
-        # unittest ones, so we could file a mypy bug to improve this.
-        six.assertRaisesRegex(self,
-                              ImportError,
-                              "Bot \"nonexistent-bot\" doesn't exists. Please "
-                              "make sure you have set up the botserverrc file correctly.",
-                              lambda: self.assert_bot_server_response(
-                                  available_bots=available_bots,
-                                  event=dict(message={'content': "@**test** test message"},
-                                             bot_email='helloworld-bot@zulip.com',
-                                             trigger='mention',
-                                             token='abcd1234'),
-                                  bots_config=bots_config))
+        # This works, but mypy still complains:
+        # error: No overload variant of "assertRaisesRegexp" of "TestCase" matches argument types
+        # [def (*args: builtins.object, **kwargs: builtins.object) -> builtins.SystemExit, builtins.str]
+        with self.assertRaisesRegexp(SystemExit,  # type: ignore
```

Comment:
I think you want assertRaisesRegex?

Reply:
I guess that isn't the problem, though; the problem is probably that `SystemExit` isn't an `Exception`.